app/views/stories/tagged_articles/index.html.erb

```diff
@@ -50,54 +50,66 @@
     <%= render "stories/tagged_articles/sidebar" %>
     <main class="articles-list" id="main-content" data-follow-button-container="true">
       <header class="flex items-center p-2 m:p-0 m:pb-2" id="on-page-nav-controls">
-        <button type="button" class="crayons-btn crayons-btn--ghost crayons-btn--icon mr-2 inline-block m:hidden" id="on-page-nav-butt-left" aria-label="<%= t("views.stories.menu.aria_left") %>">
+        <h1 class="screen-reader-only"><%= t("views.stories.heading") %></h1>
```

Comment:
Question: under what circumstances do we use a `screen-reader-only` class?

Reply:
Nvm I just read this now 😁

"This PR removes the visible <h1> element on the tagged articles feed. It isn't ideal for accessibility to be missing a clear heading level one, but we have at least introduced a "screen reader only" one."

Reply:
yeah "screen-reader-only" class is most often useful for adding some additional text that helps describe something, or a whole element that we don't want in the visible UI. We use it most often for hiding `aria-live` regions, but since we should _really_ have an `h1` on every page I'm just using it to sneak one in at least for screen reader users 🙂